addons/account/views/res_config_settings_views.xml

```diff
@@ -355,8 +355,7 @@
                                 </div>
                             </div>
                             <div class="col-12 col-lg-6 o_setting_box"
-                                id="account_batch_payment"
-                                groups="account.group_account_user">
+                                id="account_batch_payment">
```

Comment:
Can you fix the indentation and inline the div as `<div id="account_batch_payment" class="col-12 col-lg-6 o_setting_box">`